docs/gathered-data.md

```diff
@@ -979,6 +976,32 @@ gathers the alerts that have been silenced.
   - 4.10+
 
 
+## StorageCluster
+
+Collects `storageclusters.ocs.openshift.io` resources
+from Openshift Data Foundation Stack.
+
+### API Reference:
+https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+
+### Sample data
+docs/insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json
+
+### Location in archive:
+| Version | Path														|
+| ------- | --------------------------------------------------------	|
+| <= 4.11 | config/storage/{namespace}/{name}.json 					|
+| 4.12+   | config/storage/{namespace}/storageclusters/{name}.json 	|
+
+### Config ID:
+clusterconfig/storage_cluster
+
+### Since versions:
+* 4.8.49+
+* 4.10.30+
```

Comment:
4.9 is missing? Again I would double check the versions which the data should be available in.

Reply:
See the https://github.com/openshift/insights-operator/blob/master/CHANGELOG.md and the `Gather ODF config data` commit message

Reply:
The bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=2079315 talks about backporting to 4.10, but I actually can't see any cloned or related BZ for such backport.

Reply:
Well, I was actually taking as reference the comment from @jholecek-rh - so, maybe his data isn't correct?

Reply:
I would say: _trust but verify_ :smiley:

Reply:
No, my [data](https://github.com/openshift/insights-operator/pull/691#issuecomment-1289265414) is not authoritative!

* The data is based on archives that we receive. If there were no clusters with the CRs at a particular version, the version would not show up.
* The data may be skewed because of clusters that have not finished upgrading.
* The data I shared here is not even the complete output of the query that I run.

The purpose of the data was to demonstrate the discrepancy between the documentation and real data.

@tremes Mentioned he would update the script that can update the documentation with backports. I'm OK to wait for that (if it happens this year or so). :)

Reply:
OK. Can we agree that I'll adjust the data for the current release, and we can handle the documentation update/fix in other task? Otherwise this will take a very long time to get merged, and the verification doesn't seems to be easy. I already created the task for updating/validating all the gatherers documentation.

Reply:
Maybe we need to create a spreadsheet with all gathers to track down this to update the documentation.

Reply:
It's up to you. I think it would be better to have it in one PR. We don't hurry, do we? We are speaking mainly about two gatherers right now so I don't see any reason for a spreadsheet with all of them.

Reply:
Well, since we don't hurry about it, I can verify and update at least the ceph and storage documentation.

About the spreadsheet, we are going to verify and update all the gatherers information. At this point, I'm not sure if all the information that we have in our documentation is correct, and a spreadsheet where we can colaborate, comment, discuss and edit together makes IMHO a lot of diference to track down all this before we update the docs.